e2etest/l2tests/interface_selector.go

```diff
@@ -83,6 +85,40 @@ var _ = ginkgo.Describe("L2-interface selector", func() {
 			framework.ExpectNoError(err)
 		})
 
+		ginkgo.It("Validate ServiceL2Status interface", func() {
+			ginkgo.By("generate a random int to choose some interface for announcing")
+			i := rand.Intn(len(NodeNics))
```

Comment:
let's just choose the first no need for this rand (and also fail the test if len(NodeNics) == 0